Template project should have 'eslint-plugin-jest' installed as dev dependency

`interbit-template/.eslintrc` references the `jest` plugin, but it's not specified in `package.json`

In the template project, eslint errors out with: 
`Failed to load plugin jest: Cannot find module 'eslint-plugin-jest'`

Please either add a dev dependency to package.json or remove the plugin from .eslintrc

Note: This is an issue if run separately--outside of it's normal location (without Lerna)

--- a/packages/app-project/src/interbit/.eslintrc
+++ b/packages/app-project/src/interbit/.eslintrc
@@ -1,6 +1,0 @@
-{
-  "env": { "es6": false },
-  "parserOptions": {
-    "ecmaVersion": 5
-  }
-}

--- a/packages/app-todo-list/src/interbit/.eslintrc
+++ b/packages/app-todo-list/src/interbit/.eslintrc
@@ -1,6 +1,0 @@
-{
-  "env": { "es6": false },
-  "parserOptions": {
-    "ecmaVersion": 5
-  }
-}

--- a/packages/interbit-template/src/interbit/.eslintrc
+++ b/packages/interbit-template/src/interbit/.eslintrc
@@ -1,6 +1,0 @@
-{
-  "env": { "es6": false },
-  "parserOptions": {
-    "ecmaVersion": 5
-  }
-}

